### PR TITLE
Mage Axe Changes

### PR DESCRIPTION
**Pull request by ezrn**
## Describe your changes
- Improved scaling on ice prison (4+1 seconds instead of 6+0.5)
- Fissure now gives no jump
- Lightning orb activates after 1.7 seconds no matter what
- Fire blast logic revamped
- Fire blast does less fire damage, but now does base hit damage
- Cooldowns of fissure, lighting orb, and fire blast set to their mineplex versions
## Checklist before requesting a review
- [X] I have performed a self-review of my code
- [X] I have tested my changes.

### CODE REVIEW

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/axe/FireBlast.java [hunk 1]
```diff
@@ -153,50 +148,58 @@ public void onProjectileHit(ProjectileHitEvent event) {
             if (level < 1) {
                 return;
             }
+            new BukkitRunnable() {
+                @Override
+                public void run() {
+                    doExplosion(shooter, largeFireball.getLocation(), level);
+                }
+            }.runTaskLater(champions, 1L);
```
Comment (Mykindos) [inline champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/axe/FireBlast.java:156]: Use UtilServer.runTaskLater

**Review by Mykindos — CHANGES_REQUESTED** [review]

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/axe/LightningOrb.java [hunk 2]
```diff
@@ -113,43 +107,44 @@ public SkillType getType() {
...
 
     @Override
     public void onTick(ThrowableItem throwableItem) {
-        throwableItem.getLastLocation().getWorld().playSound(throwableItem.getLastLocation(), Sound.BLOCK_LAVA_EXTINGUISH, 0.6f, 1.6f);
-        throwableItem.getLastLocation().getWorld().spawnParticle(Particle.FIREWORK, throwableItem.getLastLocation(), 1);
+        if ((throwableItem.getAge() / 50) > getDelay(getLevel((Player) throwableItem.getThrower())) * 20) {
+            activateOrb((Player) throwableItem.getThrower(), throwableItem, getLevel((Player) throwableItem.getThrower()));
+            System.out.println("removed here");
+            System.out.println("age" + throwableItem.getAge());
+            System.out.println("required age: " + getDelay(getLevel((Player) throwableItem.getThrower())) * 20);
```
Comment (Mykindos) [inline champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/axe/LightningOrb.java:131]: I dont need to tell you what to do

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/axe/LightningOrb.java [hunk 3]
```diff
@@ -113,43 +107,44 @@ public SkillType getType() {
...
+    }
+
+    private void activateOrb(Player playerThrower, ThrowableItem throwableItem, int level) {
+        for (LivingEntity ent : UtilEntity.getNearbyEnemies(playerThrower, throwableItem.getItem().getLocation(), getRadius(level))) {
+            if (!throwableItem.getImmunes().contains(ent)) {
+                championsManager.getEffects().addEffect(ent, playerThrower, EffectTypes.SLOWNESS, slowStrength, (long) (getSlowDuration(level) * 1000));
+                championsManager.getEffects().addEffect(ent, EffectTypes.SHOCK, (long) (getShockDuration(level) * 1000));
+                playerThrower.getLocation().getWorld().strikeLightning(ent.getLocation());
+                UtilDamage.doCustomDamage(new CustomDamageEvent(ent, playerThrower, null, DamageCause.CUSTOM, getDamage(level), false, getName()));
+            }
```
Comment (Mykindos) [inline champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/axe/LightningOrb.java:146]: Do we want a line of sight check here? Feel like just throwing it next to someones walled in base could be really annoying

**Review by Mykindos — CHANGES_REQUESTED** [review]

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/axe/LightningOrb.java (hunk 3 above)
Comment (ezrn) [reply champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/axe/LightningOrb.java:146]: oh yeah I was gonna add that I forgot thank you

**Pull request merged**